client/src/components/Footer.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Link, useLocation } from "react-router-dom";
+import styled from "@emotion/styled";
+import AccountIcon from "../stories/assets/account-icon.svg";
+import ChatIcon from "../stories/assets/message-icon.svg";
+import SettingsIcon from "../stories/assets/settings-icon.svg";
+
+const MainFooter = styled.footer`
+  width: 100%;
+
+  background: #ade1cd;
+  border: none;
+  border-radius: 0;
+`;
+
+const Navbar = styled.nav`
+  display: flex;
+  margin: 0.2 em;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const NavImage = styled.img`
+  margin: 10px 0;
+  opacity: ${({ activeTab }) => (activeTab ? "0.5" : "1")};
```

Comment:
`activeTab` doesn't feel like a good name, because it's not a tab.
I would suggest `active`.